### DISCUSSION TITLE
How to integrate Radix Custom Theme with Shadcn theme?

### DISCUSSION THREAD

**Discussion by biapar** [shadcn-ui/ui#4585] [General]
How to integrate Radix Custom Theme with Shadcn theme?
Is there any example?
Thanks.
https://www.radix-ui.com/colors/custom

**Comment by ewgenius** [comment]
Hey @biapar, I've just made radix theme generator for shadcn/ui https://ui.ewgenius.me/shadcn-radix-colors. For now it supports only default radix colors, but I'm planning to add custom as well.

**Reply by biapar** [reply]
> Hey @biapar, I've just made radix theme generator for shadcn/ui https://ui.ewgenius.me/shadcn-radix-colors. For now it supports only default radix colors, but I'm planning to add custom as well.

Hi, very good works. It's something that need to Shadcn. How mean "Destructive Color"?  You could add the font changes...

**Comment by jparouca** [comment]
It doesn't have the same translucent effect as radix
